apps/frontend/src/validation/utils/isEmail.ts

```diff
@@ -0,0 +1,6 @@
+export default function isEmail(email: string) {
+    const emailRegex =
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
Scary! :O 

Should we not use a library for this? Not entirely sure what the convention in NAV is